ceph-multibranch-pipeline/config/definitions/ceph-multibranch-pipeline.yml

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: ceph-multibranch-pipeline
+    project-type: multibranch
+    number-to-keep: 600
+    days-to-keep: 30
```

Comment:
Should we also reduce `days-to-keep` to 15 (also set in other jobs) or leave it like it is?

Reply:
Probably the number-to-keep will always override the days-to-keep setting, but I'd personally keep as many days as possible. There's this case where a PR is opened, all checks are run by then, and after 20-30 days of discussion (but no new commits/force-pushes) it becomes merged. If that causes an issue after merging, there's no logs to find out what happened. An alternative would be to publish comments in the PR with the test results...